docs/build/build-wallets.md

```diff
@@ -36,13 +36,21 @@ post where the Treasury Proposal was discussed.
 | [Nova Wallet](https://novawallet.io/)         | [1](https://kusama.polkassembly.io/treasury/122)                                                                                                                                                    | Live              | Nova Foundation | iOS, Android | Non-custodial | Staking  |
 | [Talisman](https://talisman.xyz/)             | Received a Tip**                                                                                                                                                                                    | Live              | TalismanSociety | Browser      | Non-custodial | Staking  |
 
-> **Important**: These third-party wallets have been funded by the community through either the Polkadot or Kusama [Treasury](learn-treasury.md). 
-> **Web3 Technologies Foundation does not endorse these wallets; you should use your own due diligence in researching and using them.** The official Polkadot 
-> Support cannot provide support for issues with these wallets or other non-Parity developed wallets.
+:::caution Web3 Technologies Foundation does not endorse these wallets
 
-> **This third-party wallet was not funded by a Treasury Proposal and instead received a tip from the Treasury. Unlike a Treasury Proposal, tipping is a separate 
-> system that the council members individually participate in to collectively decide on the value of the tip. Learn more about tipping on the 
-> [Treasury Page](../learn/learn-treasury.md##tipping). 
+These third-party wallets have been funded by the community through either the Polkadot or Kusama 
+[Treasury](learn-treasury.md). You should use your own due diligence in researching and using them.
+The official Polkadot Support cannot provide support for issues with 
+these wallets or other non-Parity developed wallets.
+
+:::
+
```

Comment:
Can this note on the tip be moved above and closer to the table?

Reply:
moved the warning before the table